.github/workflows/main.yaml

```diff
@@ -153,11 +153,11 @@ jobs:
       fail-fast: false
       matrix:
         versions: [
-          { py: "3.9", weaviate: $WEAVIATE_132, grpc: "1.59.0"},
```

Comment:
Do we still want to test this specific grpc version within this matrix of versions?